src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/memcache_servers.html.twig

```diff
@@ -0,0 +1,70 @@
+{% trans_default_domain "Admin.Advparameters.Feature" %}
+
+{% block perfs_memcache_servers %}
+<div class="form-group memcache">
+    <a
+        class="btn btn-default"
+        data-toggle="collapse"
+        href="#server-form"
+        aria-expanded="false"
+        aria-controls="server-form"
+    ><i class="material-icons">add_circle</i> {{ 'Add server'|trans }}</a>
+</div>
+
+<div id="server-form" class="collapse col-md-10">
+    <div class="form-group">
+        <label class="form-control-label">{{ 'IP Address'|trans }}</label>
+        {{ form_errors(form.memcache_ip) }}
+        {{ form_widget(form.memcache_ip) }}
+    </div>
+
+    <div class="form-group">
+        <label class="form-control-label">{{ 'Port'|trans }}</label>
+        {{ form_errors(form.memcache_port) }}
+        {{ form_widget(form.memcache_port) }}
+    </div>
+
+    <div class="form-group">
+        <label class="form-control-label">{{ 'Weight'|trans }}</label>
+        {{ form_errors(form.memcache_weight) }}
+        {{ form_widget(form.memcache_weight) }}
+    </div>
+
+    <div class="form-group">
+        <div class="float-right">
+            <a href="#" onclick="app.addServer();" class="btn btn-primary">{{ 'Add Server'|trans }}</a>
+            <a href="#" onclick="app.testServer();" class="btn btn-primary">{{ 'Test Server'|trans }}</a>
+        </div>
+    </div>
+</div>
+
+<div id="servers-list" class="memcache">
+    <div class="form-group">
+        <table class="table" id="servers-table">
+            <thead>
+            <tr>
+                <th class="fixed-width-xs"><span class="title_box">{{ 'ID'|trans({}, 'Admin.Global') }}</span></th>
+                <th><span class="title_box">{{ 'IP address'|trans({}, 'messages') }}</span></th>
```

Comment:
Never use messages as a domain. Should be"Admin.Advparameters.Feature".

Reply:
I agree too, but it's the actual key :) to be done in another - and translation-specific topic - pull request :+1: